core/trino-main/src/main/java/io/trino/server/security/PasswordAuthenticatorManager.java

```diff
@@ -60,27 +62,31 @@ public boolean isLoaded()
     }
 
     public void loadPasswordAuthenticator()
-            throws Exception
     {
         if (!required.get()) {
             return;
         }
 
         File configFile = CONFIG_FILE.getAbsoluteFile();
-        Map<String, String> properties = new HashMap<>(loadPropertiesFrom(configFile.getPath()));
+        try {
+            Map<String, String> properties = new HashMap<>(loadPropertiesFrom(configFile.getPath()));
```

Comment:
Can you please only wrap `loadPropertiesFrom` with `try-catch`? Maybe add private method.